orthoseg/postprocess.py

```diff
@@ -122,6 +122,8 @@ def postprocess(config_path: Path, config_overrules: List[str] = []):
         # Prepare some parameters for the postprocessing
         nb_parallel = conf.general.getint("nb_parallel")
 
+        keep_original_file = conf.postprocess.getboolean("keep_original_file")
```

Comment:
```suggestion
        keep_original_file = conf.postprocess.getboolean("keep_original_file", True)
```
Safest to specify a default value. Otherwise the default will be `None` if the parameter is not specified in a config file, which might lead to odd behaviour.